Esterni/PianoProgetto/components/Pianificazione.tex

```diff
@@ -0,0 +1,207 @@
+	\item\textbf{Glossario:} si individuano tutti i termini che possono risultare ambigui e vengono affiancati da una definizione all'interno del documento;  
+	\item \textbf{Lettera di Presentazione:} Documento in cui il gruppo \Gruppo si candida come fornitore del prodotto software richiesto.
+\end{itemize}
+\subsubsection{Fasi}
+la pianificazione di questo periodo è stata suddivisa nelle seguenti sotto-fasi
+\begin{itemize}
+	\item \textbf{dal 2020-11-09 al 2020-11-30:}
+	\begin{itemize}
+	\item vengono prese decisioni riguardanti: il nome del team, l'indirizzo e-mail, il logo, la frequenza degli incontri, gli strumenti per la comunicazione;
+	\item viene svolta la discussione sui capitolati, esponendo le preferenze personali dei membri del gruppo. Viene dunque analizzato ogni capitolato, ponendo attenzione ai pro e contro di ogni capitolato e viene fatta una breve valutazione rischi per ognuno. Successivamente si può dunque iniziare la stesura dello Studio di Fattibilità;
+	\item vengono stesi i verbali interni relativi alle riunioni del gruppo durante questa sotto-fase.
+	\end{itemize}
+	\item \textbf{dal 2020-12-01 al 2020-12-21:}
+	\begin{itemize}
+		\item viene iniziata la stesura delle \textit{Norme di Progetto} per fissare le regole delle attività del gruppo;
+		\item inizio della stesura del \textit{Piano di Progetto}, dove viene descritta la pianificazione del lavoro da svolgere e la suddivisione dei ruoli tra i membri del gruppo;
+		\item inizio stesura del \textit{Glossario}, registrando i termini usati nei documenti che risultano ambigui;
+		\item stesi i verbali interni relativi agli incontri svoltesi durante questa sotto-fase.
+	\end{itemize}
+	\item \textbf{dal 2020-12-22 al 2021-01-07:}
+	\begin{itemize}
+		\item il gruppo inizia ad analizzare il capitolato scelto, ricercando i requisiti e procedendo con la stesura dell'\textit{Analisi dei Requisiti};
+		\item Si procede con la stesura del \textit{Piano di Qualifica}, individuando i metodi per garantire la qualità del prodotto;
+		\item stesura della \textit{Lettera di Presentazione};
+		\item Stesi i verbali interni relativi agli incontri svoltesi durante questa sotto-fase.
+	\end{itemize}
+	\item \textbf{dal 2021-01-08 al 2021-01-11 :}
+	\begin{itemize}
+		\item viene completata la stesura del \textit{Glossario};
+		\item il gruppo uniforma tutti i documenti prodotti alle regole stabilite nelle \textit{Norme di Progetto} se necessario;
+		\item vengono svolte le ultime attività di verifica sui documenti.
+	\end{itemize}
+\end{itemize}
+
+\subsubsection{Diagramma di Gantt: analisi}
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale = 0.25]{components/img/Analisi.jpg}
+    \caption{Diagramma di Gantt della fase di Analisi}
+    \label{fig:Diagramma di Gantt, fase di Analisi}
+\end{figure}
+
+\newpage
+\subsection{Consolidamento dei Requisiti}
```

Comment:
C'è da discutere su questo riguardo, se si ha un consolidamento dei requisiti viene naturale che ci dovrebbe essere un consolidamento di anche tutti le altre fasi, forse è meglio accorparlo con un'altra fase in modo da non avere un errore logico.

Reply:
In generale la pianificazione forse è da ripensare per colpa di questa cosa